### ISSUE TITLE
Improvement: "Migration is underway" /sites info not accurate for assisted migrations

### ISSUE THREAD

**Issue by cuemarie** [Automattic/wp-calypso#97667]
### TLDR

Update the "Migration is underway" page in `/sites` so that it doesn't indicate that their assisted migration has started before it has.

### Details

While testing the migration flow today, I requested an assisted migration (for a non-wp site).  While the `We'll take it from here` page notes that I will hear back from support with information on how to provide my credentials (which is correct), the `/sites` dashboard information tells me the migration is already underway, and that I should sit back, and I will hear back in 2-3 days (which is often incorrect).

![Image](https://github.com/user-attachments/assets/cdc71a7b-771d-4b2c-9eb9-f6999f4c53de)

### Why

There are many cases where we can't start immediately on the migration (user provided wrong credentials or backup files, we don't have the right permissions on the site, the site is ecommerce and needs to be scheduled, its a multisite, or not even a wp site at all), and this notice may inadvertently overpromise what we can deliver, and discourage customers from checking their email right away, which we frequently need them to do _so that we can access their site and get started._

### How

When a customer requests an assisted migration, could this page instead say `Migration Assistance Requested`, with detail that start with how we'll be reaching out to them soon with instructions on how to grant us access to the source site? 

Bonus: Could this page show a button to "Cancel request", and could that notify us somehow? (This might generate double the tickets and not actually prevent us from replying, so not sure about what's possible there :) )

**Comment by Robertght** [comment]
Setting a low priority on this one and marking it as Triaged on the One Board.

**Comment by gabrielcaires** [comment]
@fditrapani What do you think we can do? maybe update the copy?

**Comment by fditrapani** [comment]
Thanks for the ping @gabrielcaires. One option would be to create a new status to adjust expectations as @cuemarie suggested. Initially, we'd set Migration requested and update the copy as to what can be expected. Once the migration starts, then we update the status and show the content we're currently showing. 

Alternatively, on the lower effort side, we can just update the copy to better reflect the expectation. We can use the title `Migration Assistance Requested` and adjust the timeline to what would be more realistic.

**Comment by cuemarie** [comment]
> Alternatively, on the lower effort side, we can just update the copy to better reflect the expectation. We can use the title Migration Assistance Requested and adjust the timeline to what would be more realistic.

The technical migration only really lasts 30 minutes on average; at the end of that, this page would be removed with a "migration-completed" update, and we'd email them back to ask them to look things over and then help with their domain connection.  I don't think updating the copy of the page when we're actively working on things would be worth the effort.

**Comment by cuemarie** [comment]
> Bonus: Could this page show a button to "Cancel request", and could that notify us somehow? (This might generate double the tickets and not actually prevent us from replying, so not sure about what's possible there :) )

I broke this out into its own feature request, here: https://github.com/Automattic/wp-calypso/issues/99065

**Comment by cuemarie** [comment]
Hi @fditrapani ! Any updates on this?  Here's a great example of the confusion for both customers and HEs that can stem from that screen:

p1738614516666679-slack-C06PZC1RR5F

**Comment by cuemarie** [comment]
Another example: https://a8c.zendesk.com/agent/tickets/9359026

Customer has not replied to 3 separate emails where we've asked them to send us their credentials so we can get started, but left feedback on this ticket saying:

>  I requested migrating a website to Wordpress.com last Monday and am worried that it is taking so long.

The "Your migration is underway" screen is definitely incorrect in this setting, and may be contributing to their ignoring our emails and just waiting for the migration to finish.  Instead, it could be instructing customers that _they need to reply to us_, helping improve this sticking point.

(I cleared the migration status on the site using `clear migration flow stickers` in their [BRC](https://mc.a8c.com/tools/reportcard/blog/?blog_id=241048191) for now, so they hopefully realize things are not yet underway, and get back to us.

**Comment by cuemarie** [comment]
Updated the priority on this as this is a pretty important point in the customer's experience of an assisted migration.

**Comment by fditrapani** [comment]
> Hi @fditrapani ! Any updates on this? Here's a great example of the confusion for both customers and HEs that can stem from that screen:

Thanks for the ping @cuemarie. @gabrielcaires might be able to share an update on this as him and his team would be working on it. I'm happy to provide feedback on any suggestions or review any work.

**Comment by markbiek** [comment]
I'm leaving this open so I can make the copy changes to this screen as well:

https://github.com/Automattic/wp-calypso/pull/100786#pullrequestreview-2664538986

![Image](https://github.com/user-attachments/assets/18d04e5b-23a0-49fc-9094-23ae11f93e01)

**Comment by daledupreez** [comment]
Note that this issue is closely related to https://github.com/Automattic/wp-calypso/issues/101017, which proposes that we consolidate the two sets of details on these screens into the site-specific view.
